src/client.js

```diff
@@ -73,6 +75,14 @@ export default class Client {
   }
 
   addFile(file) {
-    this.uppy.addFile({ name: file.name, type: file.type, data: file });
+    this.uppy.addFile({ name: file.name, type: file.type, data: this.updateLastModified(file) });
+  }
+
+  /* eslint-disable-next-line class-methods-use-this */
+  updateLastModified(file) {
```

Comment:
Use a simple function here instead of a class method. This way you can also get rid of the eslint comment.

Reply:
Yes I thought so too, but how can I test it if it's not a class method?

Reply:
Export it 😄

Reply:
Ok... can do that. That's like a well practised pattern, right?

Reply:
you could also make a utils module and put the function there.

Reply:
I put the function in the utils module, somehow seemed "nicer" to me 😀